bomba/roles/hivemq-mqtt-web-client/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Clone hivemq-mqtt-web-client
+  git:
+    repo: "https://github.com/bitraf/hivemq-mqtt-web-client"
+    dest: "/var/www/mqtt.bitraf.no/web"
+    version: bitraf
```

Comment:
hva med å kalle grenen `live`, `ansible` eller noe for å indikere at den blir automatisk tatt i bruk?